content/news/2024-02-03-bevy-0.13/index.md

```diff
@@ -64,7 +64,20 @@ Some use cases for primitive shapes include meshing, gizmos, bounding volumes, c
 [Primitive RFC]: https://github.com/bevyengine/rfcs/blob/main/rfcs/12-primitive-shapes.md
 [collection of primitives]: https://dev-docs.bevyengine.org/bevy/math/primitives/index.html
 
-### Meshing
+### Rendering
+
+Now we're taking a closer look at how to visualize the primitives both in 2D
+and in 3D spaces. In this section, we delve into the rendering process with
+meshes and gizmos. You can check out at all supported rendering capabilities
+that are available by taking a look at the following linked example.
+
+![On the left: A cuboid rendered with gizmos. It consists of 12 white lines. On the right: A cuboid rendered with meshes. It consists of 6 white faces.](cuboids.png)
+
+![On the left: A torus rendered with gizmos. It consists of many small rings, all connected by 4 big rings. On the right: A torus rendered with meshes. A shape that looks like a donut.](tori.png)
```

Comment:
These screenshots have this slight black border. It doesn't look good on the website because there the corners are rounded.

![kuva](https://github.com/Jondolf/bevy-website/assets/57632562/19d170a0-fd8a-4b0d-95d2-ae20fd71dc6f)

Reply:
Fixed and also resized the images to have the same dimensions